src/vmm/src/devices/virtio/queue.rs

```diff
@@ -1128,11 +1117,11 @@ mod verification {
 
     #[kani::proof]
     #[kani::unwind(0)]
-    fn verify_actual_size() {
+    fn verify_size() {
         let ProofContext(queue, _) = kani::any();
 
-        assert!(queue.actual_size() <= queue.get_max_size());
-        assert!(queue.actual_size() <= queue.size);
+        assert!(queue.size <= queue.max_size);
+        assert!(queue.size <= queue.size);
     }
```

Comment:
the second assertion is very useless now, but actually the entire harness can be deleted, as it just tests the mock now. But I'll handle this in a follow up, no need to fix here